man/tpm2_loadexternal.1.md

```diff
@@ -19,11 +19,13 @@ TPM. The command allows loading of a public area or both a public and a
 sensitive area.
 
 # Output
-The tool outputs the handle and name of the loaded object in a YAML format.
+The tool outputs the name of the loaded object in a YAML format and saves a
+context file for future interactions with the object. The context file name
+defaults to *external.ctx* and can be specified with **-o**.
```

Comment:
object.ctx

The problem will be clobbering something that's there.... this even occurs in the current one if they don't specify the name. Perhaps a routine that looks at what's there and does something like:
```
object.ctx
object1.ctx
object2.ctx
```

Reply:
Great idea, I've got this mostly coded up but will do some more testing in the new year before I push here.

Reply:
Implemented in fc6484a